tests/session/session-getTransactionOptions-001.phpt

```diff
@@ -0,0 +1,60 @@
+--TEST--
+MongoDB\Driver\Session::getTransactionOptions()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto(); ?>
+<?php skip_if_no_transactions(); ?>
```

Comment:
I don't believe `startTransaction` and `abortTransaction` will communicate with the server, so we shouldn't need to require transactions.

Instead, we should just need some SKIPIFs for session support (both client and server-side):

```
<?php skip_if_not_libmongoc_crypto(); ?>
<?php skip_if_not_replica_set_or_mongos_with_replica_set(); ?>
<?php skip_if_server_version('<', '3.6'); ?>
```

While looking into this, I realized that some session functional tests don't actually require a replica set. For instance, [session-003.phpt](https://github.com/mongodb/mongo-php-driver/blob/master/tests/session/session-003.phpt) uses:

```
<?php skip_if_not_libmongoc_crypto(); ?>
<?php skip_if_not_live(); ?>
<?php skip_if_server_version('<', '3.6'); ?>
```

This appears to be OK as standalones still report [`logicalSessionTimeoutMinutes` in `isMaster` responses](https://docs.mongodb.com/manual/reference/command/isMaster/#isMaster.logicalSessionTimeoutMinutes), and that's all libmongoc requires to check for server-side session support. Our tests that actually pass the session in a server command _do_ require a replica set, as a standalone would surely error out if it received an `lsid` command option.

With that in mind, you can consider using the more lax SKIPIFs here and seeing if this runs on a 3.6 standalone. If so, I'll defer to you if you'd rather keep the requirements as permissive as possible to expand our test coverage (e.g. perhaps we have some test environments that are just using standalones).